app/styles/_kve.less

```diff
@@ -192,6 +192,12 @@
   margin-bottom: 0;
   padding-right: 5px;
   width: 50%;
+  &.prefix-header {
+    display: none;
```

Comment:
If the header is totally hidden, I'm not sure how you know what that field does on mobile.

Reply:
well when empty it does display `placeholder="Add prefix"` :)
I'll append it to the single header and show at mobile.

<img width="381" alt="screen shot 2017-10-30 at 5 24 10 pm" src="https://user-images.githubusercontent.com/1874151/32196486-33ad414a-bd97-11e7-9927-d2026bd15f16.png">